src/cmd/tools/dtest/docker/harness/resources/coordinator.go

```diff
@@ -323,6 +335,77 @@ func (c *coordinator) WriteCarbon(
 	// return nil
 }
 
+func (c *coordinator) WriteProm(name string, tags map[string]string, samples []Sample) error {
+	if c.resource.closed {
+		return errClosed
+	}
+
+	var (
+		url = c.resource.getURL(7201, "api/v1/prom/remote/write")
+
+		reqLabels  = make([]prompb.Label, 0)
+		reqSamples = make([]prompb.Sample, 0)
+	)
+
+	reqLabels = append(reqLabels, prompb.Label{
+		Name:  []byte(model.MetricNameLabel),
+		Value: []byte(name),
+	})
```

Comment:
Micro nit (slightly less syntax for the eyes to parse):
```suggestion
		reqLabels = []prompb.Label{{Name: []byte(model.MetricNameLabel), Value: []byte(name)}}
		reqSamples  []prompb.Sample
```
or something in that direction.

Reply:
https://github.com/m3db/m3/pull/3065/commits/de8c17f6e067c6426c7e18d7b620c8b8967a62d4#diff-36f3f4899036050c43cca0b378f19429cc8ceb118ebffd0c42f672ee13d507e8L344-L353